### PR TITLE
Challenge/equipo4/amazux

### PR DESCRIPTION
**Pull request by fredyzz**

### CODE REVIEW

File: 00-TEAMS/04-TEAM/amazux/src/components/Cart/CartList/index.jsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { deleteProduct } from '../../../redux/actions/actionCreators';
+
+const CartList = ({ cart, dispatch }) => {
+  const handleClick = (element) => {
+    dispatch(deleteProduct(element));
+  };
+  return (
+
```
Comment (ulldecorb) [inline 00-TEAMS/04-TEAM/amazux/src/components/Cart/CartList/index.jsx:12]: Delete all empty line breaks

File: 00-TEAMS/04-TEAM/amazux/src/components/Cart/index.jsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import CartHeader from './CartHeader/index';
+import CartList from './CartList/index';
+
+const Cart = () => (
+
```
Comment (ulldecorb) [inline 00-TEAMS/04-TEAM/amazux/src/components/Cart/index.jsx:6]: More empty line breaks to delete

File: 00-TEAMS/04-TEAM/amazux/src/components/Commons/FeaturedProducts/featuredProducts.scss [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+            max-width: 100%;
+            height: auto;
+        }
+        a {
+            text-decoration: none;
+            color: black;
+            cursor: pointer;
+        }
+    }
+
```
Comment (ulldecorb) [inline 00-TEAMS/04-TEAM/amazux/src/components/Commons/FeaturedProducts/featuredProducts.scss:25]: More empty line breaks to delete

File: 00-TEAMS/04-TEAM/amazux/src/components/Commons/FeaturedProducts/index.jsx [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+                  <h3>{product.name}</h3>
+                  <img src={product.img.url} alt={product.name} />
+                  <p>
+                    {`${product.cost} €`}
+                  </p>
+                </Link>
+              </div>
+            ))}
+          </div>
+
```
Comment (ulldecorb) [inline 00-TEAMS/04-TEAM/amazux/src/components/Commons/FeaturedProducts/index.jsx:25]: More empty line breaks to delete

File: 00-TEAMS/04-TEAM/amazux/src/components/Commons/Loader/index.jsx [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+const Loader = () => (
+  <div className="loader-container">
+    <div className="lds-ring">
+      <div />
+      <div />
+      <div />
+      <div />
+    </div>
+  </div>
+
```
Comment (ulldecorb) [inline 00-TEAMS/04-TEAM/amazux/src/components/Commons/Loader/index.jsx:13]: More empty line breaks to delete

File: 00-TEAMS/04-TEAM/amazux/src/components/Commons/ProductsList/productList.scss [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+    &__img {
+        width: 140px;
+    }
+    
+    &__details {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+    }
+    a {
```
Comment (ulldecorb) [inline 00-TEAMS/04-TEAM/amazux/src/components/Commons/ProductsList/productList.scss:26]: Add break line

**Review by ulldecorb — APPROVED** [review]
A few line breaks to correct

**Review by cristian-bars — APPROVED** [review]

**Review by annasm07 — APPROVED** [review]

**Review by IsabelGomez87 — APPROVED** [review]

**Pull request merged**